src/DetailsView/details-view-container.tsx

```diff
@@ -173,7 +174,7 @@ export class DetailsViewContainer extends React.Component<DetailsViewContainerPr
         const issueTrackerPath =
             (storeState.userConfigurationStoreData.bugServicePropertiesMap &&
                 storeState.userConfigurationStoreData.bugServicePropertiesMap.gitHub &&
-                storeState.userConfigurationStoreData.bugServicePropertiesMap.gitHub.repository) ||
+                (storeState.userConfigurationStoreData.bugServicePropertiesMap.gitHub as GitHubBugFilingSettings).repository) ||
```

Comment:
This should be changed to pass the whole settings object right. I'm assuming u are not doing it in this PR?

Reply:
That's correct; leaving as is for now in this PR.